### PR TITLE
Force unicode run/decode

### PR DESCRIPTION
**Pull request by jupierce**

### CODE REVIEW

**Comment by jupierce** [comment]
fyi @thiagoalessio (but not until you are back from PTO!)

File: doozerlib/cli/__main__.py [hunk 1]
```diff
@@ -2192,7 +2192,17 @@ def update_operator_metadata(runtime, nvr_list, stream, image_ref_mode, merge_br
         # merge_branch = runtime.group_config.operator_metadata_branch
 
     try:
-        ok = ThreadPool(cpu_count()).map(operator_metadata.update_and_build, [
+
+        def gotta_catchem_all(*args, **kwargs):
```
Comment (sosiouxme) [inline doozerlib/cli/__main__.py]: :rofl:

Comment (sosiouxme) [reply doozerlib/cli/__main__.py]: but we really do need this

File: doozerlib/operator_metadata.py [hunk 2]
```diff
@@ -229,7 +229,8 @@ def rename_csv_yaml_filename_of_arch_copy(self, arch):
             '{}.0.'.format(self.channel),          # e.g. 4.2.0
             '{}.0-{}.'.format(self.channel, arch)  # e.g. 4.2.0-s390x
         )
-        exectools.cmd_assert('mv {} {}'.format(original_filename, new_filename))
+        if original_filename != new_filename:
```
Comment (sosiouxme) [inline doozerlib/operator_metadata.py]: hm. ok that'll work but... i'd rather just rip out the renaming

**Pull request merged**